content/graphing/infrastructure/livecontainers.md

```diff
@@ -95,6 +95,16 @@ It would be easy to pivot by ECS `ecs_task_name` and `ecs_task_version` and unde
 
 {{< img src="graphing/infrastructure/livecontainers/tasksxversion.png" alt="Tasks x version" responsive="true" style="width:80%;">}}
 
+## Scatterplot for Live Containers
+
+The scatterplot is designed for high volume data analysis and, with intelligent grouping, can help you explore all of the containers across your entire environment. With this new visualization, you are able to compare these metrics with one another to better understand the performance of your containers.
+
+In order to access the scatterplot graph in the Containers page, click on the ‘Show Summary graph’ button next to the ‘Group by tag’ text field. Finally, click on the Scatterplot tab to switch from the Timeseries graph view.
```

Comment:
r / `click on the ‘Show Summary graph’ button next to the ‘Group by tag’ text field. Finally, click on the Scatterplot tab to switch from the Timeseries graph view. ` / `click on the *Show Summary graph* button next to the *Group by tag* text field then click on the *Scatterplot* tab`